<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDFDialog</class>
 <widget class="QDialog" name="SDFDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>459</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SDF Import</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblFilename">
         <property name="text">
          <string>Filename:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="txtFilename">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;</string>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblContent">
         <property name="text">
          <string>Content:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="txtContent">
         <property name="text">
          <string>N/A</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="qqIgnoreMissingValues">
       <property name="text">
        <string>Ignore missing required values (SDF Parser)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="qqCollisionLinksHidden">
       <property name="text">
        <string>Assign collision links to layer 9 </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="qqJointsHidden">
       <property name="text">
        <string>Assign joints to layer 10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="qqConvexDecompose">
       <property name="text">
        <string>Convex decompose non-convex collision links</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="qqConvexDecomposeDlg">
       <property name="text">
        <string>Show convex decomposition dialog</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="qqCreateVisualLinks">
       <property name="text">
        <string>Create visual links if none</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="qqCenterModel">
       <property name="text">
        <string>Center model above ground</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="qqModelDefinition">
       <property name="text">
        <string>Prepare model definition if feasable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="qqAlternateMasks">
       <property name="text">
        <string>Alternate local respondable masks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="qqPositionCtrl">
       <property name="text">
        <string>Enable position control for revolute and prismatic joints</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="qqImport">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>If you experience a sudden crash during the import operation, make sure to enable 'Show convex decomposition dialog' and adjust the parameters there. Another option would be to simply disable 'Convex decompose non-convex collision links'.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>qqCollisionLinksHidden</tabstop>
  <tabstop>qqJointsHidden</tabstop>
  <tabstop>qqConvexDecomposeDlg</tabstop>
  <tabstop>qqCreateVisualLinks</tabstop>
  <tabstop>qqCenterModel</tabstop>
  <tabstop>qqModelDefinition</tabstop>
  <tabstop>qqAlternateMasks</tabstop>
  <tabstop>qqImport</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
